ecs-init/go.mod

```diff
@@ -1,10 +1,16 @@
 module github.com/aws/amazon-ecs-agent/ecs-init
 
-go 1.19
+go 1.21
+
+toolchain go1.22.2
```

Comment:
What is the use case for having a toolchain version with a different minor version than the module's go version here? (i.e., 22 vs 21)

Reply:
No need for different versions. I updated my go version and the toolchain directive was automatically added at some point.

Reply:
Got it, I see this line has now been removed in most recent commit. Thanks!